core/build_playerglobal/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+//! An internal Ruffle utility to build our playerglobal
+//! `library.swf`
+
+use std::collections::HashSet;
+use std::fs::File;
+use std::path::PathBuf;
+use std::process::Command;
+use swf::DoAbc;
+use swf::Header;
+use swf::SwfStr;
+use swf::Tag;
+use walkdir::WalkDir;
+
+/// If successful, returns a list of paths that were used. If this is run
+/// from a build script, these paths should be printed with
+/// cargo:rerun-if-changed
+pub fn build_playerglobal(
+    repo_root: PathBuf,
+    out_dir: PathBuf,
+) -> Result<Vec<PathBuf>, Box<dyn std::error::Error>> {
+    let classes_dir = repo_root.join("core/src/avm2/globals/");
+    let asc_path = repo_root.join("core/build_playerglobal/asc.jar");
+
+    let out_path = out_dir.join("playerglobal.swf");
+
+    let mut classes = Vec::new();
+    for entry in WalkDir::new(&classes_dir) {
+        let entry = entry?;
+        if entry.path().extension().and_then(|e| e.to_str()) == Some("as") {
+            classes.push(entry.into_path());
+        }
+    }
+
+    // These classes are currently stubs - they're referenced by
+    // other classes that we need to compile, but the real definition
+    // is in Ruffle itself (in Rust code).
+    // As a result, we don't emit them into the final SWF (but we do
+    // provide them to asc.jar with '-import' to link against).
+    let stub_classes: HashSet<_> = ["Object", "Number", "Boolean", "String"]
```

Comment:
Can't this be simplified as shown in https://doc.rust-lang.org/std/collections/struct.HashSet.html#method.from?
```rs
let stub_classes: HashSet<_> = ["Object", "Number", "Boolean", "String"].into();
```